logic/multisignature.js

```diff
@@ -274,21 +287,24 @@ Multisignature.prototype.verify = function(transaction, sender, cb) {
 
 /**
  * Returns transaction with setImmediate.
- * @param {transaction} transaction
- * @param {account} sender
+ *
+ * @param {transaction} transaction - Description of the param
+ * @param {account} sender - Description of the param
  * @param {function} cb - Callback function.
  * @returns {setImmediateCallback} Null error
  * @todo check extra parameter sender.
+ * @todo Add descriptions for the params
  */
 Multisignature.prototype.process = function(transaction, sender, cb) {
 	return setImmediate(cb, null, transaction);
 };
 
 /**
  * Returns a buffer with bytes from transaction asset information.
+ *
  * @requires bytebuffer
  * @see {@link https://github.com/dcodeIO/bytebuffer.js/wiki/API}
- * @param {transaction} transaction - Uses multisignature from asset.
+ * @param {transaction} transaction - Uses multisignature from asset
  * @returns {!Array} Contents as an ArrayBuffer.
```

Comment:
From JSDoc documentation: `Indicates that the value is of the specified type, but cannot be null.`

Reply:
Right, but that's presumably the case for a lot of our types in other cases. So far we haven't established a consistent pattern for nullable/optional/non-nullable types.

Reply:
Sure, but in the cases where we provide this additional information, it doesn't really hurt, or not?

In my opinion, we should not use `{?number}`, because this is kind of the default, but if we want to specify, that a value cannot be null, `{!number}` can be a good way to be more precise